lib/trento/users/user.ex

```diff
@@ -86,6 +87,7 @@ defmodule Trento.Users.User do
     |> validate_password()
     |> custom_fields_changeset(attrs)
     |> cast(attrs, [:password_change_requested_at])
+    |> cast(attrs, [:analytics_opt_in])
```

Comment:
We can put the new field together with the previous one, no need to add a new `cast`

Reply:
This has been resolved. Thanks! 😄